docs/support.md

```diff
@@ -1,6 +1,6 @@
 # Support
 
 Best way to ask general questions about documentation or particular use cases is to [create a GitHub issue](https://github.com/cirruslabs/cirrus-ci-com/issues/new).
-Support team will answer it by replying or updating documentation.
+Our support team will answer it by replying or updating documentation. They tend to respond within 48 hours.
```

Comment:
There is no SLA claimed even though responses are usually send within hours. Let's not claim it right now.